linux/src/main.cpp

```diff
@@ -329,6 +329,10 @@ int main(int argc, char* argv[]) {
         }
     }
 
+    finishUrlRequests();
+
+    Tangram::shutdown();
```

Comment:
There is no hard having an explicit control on destroying individual tangram blocks!